pkg/resource/agents-toggle/observability_bundle_configmap.go

```diff
@@ -1,4 +1,4 @@
-package loggingagentstoggle
+package agentstoggle
```

Comment:
You probably need to update main.go or the controller with that change :D

Reply:
Done :)